app/controllers/container_controller.rb

```diff
@@ -95,6 +95,7 @@ def show_list
   end
 
   def explorer
+    process_show_list
```

Comment:
1. I think the right place for setting the breadcrumbs is after (or immediately bofore) we actually  render the page.
punting the breadcrumbs near line 117-120 make more sense to me.

2. is it possible to use `drop_breadcrumb`,  `process_show_list` looks like an overkill ?
`drop_breadcrumb(:name => _("Container explorer"), :url => "/ems_container")`